app/assets/stylesheets/components/_organizations.scss

```diff
@@ -0,0 +1,32 @@
+.organization-details {
+  .section {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets